README.md

```diff
@@ -24,8 +22,7 @@ Note that, at minimum, the `MGLMapboxAccessToken` must be set in order to use th
 
 ### Setting Connection Type
 
-By default, the app will attempt to connect via an `iAP` connection after launch. An `iAP` connection allows you to connect to production or test hardware using a USB cord or Bluetooth. 
-If you would like to test with an emulator you will need to configure a `TCP` (i.e. WiFi) connection. Navigate to the `SDLAppConstants.swift` file and change the `connectionType` to `.tcp`. Additionally, you will need to set the correct `ipAddress` and `port` values as well.
+By default, the app will attempt to connect via an `iAP` connection after launch. An `iAP` connection allows you to connect to production or test hardware using a USB cord or Bluetooth. If you would like to test with an emulator you will need to configure a `TCP` (i.e. WiFi) connection. Navigate to the `SDLAppConstants.swift` file and change the `connectionType` to `.tcp`. Additionally, you will need to set the correct `ipAddress` and `port` values as well.
```

Comment:
The default connection type is still TCP currently.

Reply:
It's not on master anymore